reana_client/cli/utils.py

```diff
@@ -127,7 +127,34 @@ def parse_parameters(_filter):
         )
 
 
-def filter_data(parsed_filters, headers, tablib_data):
+def parse_filter_parameters(filters):
+    """Return parsed filter parameters."""
+    try:
+        status_filters = []
+        search_filters = None
+        filters = " ".join(filters).split(",")
+        for item in filters:
+            if "=" in item:
+                filter_name = item.split("=")[0].lower()
+                filter_value = item.split("=")[1]
+                if filter_name == "status":
+                    status_filters.append(filter_value)
+                elif filter_name == "name":
+                    search_filters = filter_value
+                else:
+                    click.echo("Filter {} is not supported.".format(filter_name))
```

Comment:
Maybe we can add `Error: ` prefix as we have in other places and print it in red, WDYT?

Reply:
I can add but this is not an error, just info for a wrong input. Do we have some color for `info` as well, that would be more apt here?

Reply:
I don't think so.. we just use white for info messages. Nevermind, it's not very important as long as we inform the user.

Reply:
Yes, as we do for the `logs` command already:

```console
$ reana-client logs -w helloworld-yadage --filter staaatus=foo
Error: filter 'staaatus' is not valid.
Available filters are 'compute_backend' 'docker_img' 'status' 'step'.
```

We can unify the formatting at least for the various `--filter` commands everywhere.

BTW note that the output message formatting cosmetics will remain elsewhere to be amend as well, for example in `ping` we don't report it either:

```console
$ reana-client ping
REANA client is not connected to any REANA cluster.
```

Something for another issue. We have quite a potential to improve cosmetics regarding `==> ERROR: ...` and friends.

Reply:
Created a new issue #431